haiku-apps/ppviewer/ppviewer-1.0.0.recipe

```diff
@@ -25,6 +25,10 @@ BUILD_PREREQUIRES="
 	cmd:g++
 	"
 
+if [ "$effectiveTargetArchitecture" == "x86_64" ] ; then
+PATCHES="ppviewer-1.0.0.patchset"
```

Comment:
We like to use `$portVersion` when that's possible :)
```diff
-PATCHES="ppviewer-1.0.0.patchset"
+PATCHES="ppviewer-$portVersion.patchset"
```
(you can do the same change on ther other recipes you are already editing.)